.github/workflows/deploy-pages.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy pages
+
+on:
+  push:
+    branches: ["pages-only"]
+
+  workflow_dispatch:
```

Comment:
What's this?

Reply:
this allows running workflow [manually](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow#configuring-a-workflow-to-run-manually), I notice it must add the workflow in default branch (main), I removed it